### PR TITLE
glimpse: Drop package and plugins

### PR DESCRIPTION
**Pull request by onny**
###### Description of changes

Drop package and plugins since project is discontinued and doesn't build anymore https://glimpse-editor.org/posts/a-project-on-hiatus/

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/chemistry/d-seams/default.nix [hunk 1]
```diff
@@ -1,7 +1,7 @@
-{ clangStdenv, fetchFromGitHub, catch2, rang, fmt, libyamlcpp, cmake
+{ clang9Stdenv, fetchFromGitHub, catch2, rang, fmt, libyamlcpp, cmake
 , eigen, lua, luaPackages, liblapack, blas, lib, boost, gsl }:
 
-clangStdenv.mkDerivation rec {
+clang9Stdenv.mkDerivation rec {
```
[the remark this answers is not in the document]
Comment (onny) [reply pkgs/applications/science/chemistry/d-seams/default.nix]: Ups this was a mistake :) Thank you for noticing

**Comment by onny** [comment]
Okay I removed the package and plugins :)

**Comment by erictapen** [comment]
Thanks, makes sense to drop it at this point. Tried to fix the build failure the last days, but couldn't really find the time for it.

If anybody wants to bring the package back in the future, I'd be glad to help!

**Comment by erictapen** [comment]
@GrahamcOfBorg build gimp

**Comment by SuperSandro2000** [comment]
We should have added an alias :)

**Comment by erictapen** [comment]
Oh right, I'll add one.

**Comment by erictapen** [comment]
Fixed in https://github.com/NixOS/nixpkgs/pull/181111

**Pull request merged**